csharp/Ice/hello/server/Hello.cs

```diff
@@ -7,19 +7,13 @@ namespace Demo
 {
     public class Hello : IHello
     {
-        public void SayHello(int delay, Current current)
+        public string SayHi(Current current, global::System.Threading.CancellationToken cancel)
```

Comment:
We should add a using `using System.Threading` and simplify the names in the operation signatures

```suggestion
        public string SayHi(Current current, CancellationToken cancel)
```